src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -231,7 +239,7 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
 #pragma warning restore
         {"DocumentOutlineOptions_EnableDocumentOutline", new FeatureFlagStorage(@"Roslyn.DocumentOutline")},
         {"dotnet_code_quality_unused_parameters", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.UnusedParametersPreference")},
-        {"dotnet_remove_unnecessary_suppression_exclusions", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.RemoveUnnecessarySuppressionExclusions")},
+        {"dotnet_remove_unnecessary_suppression_exclusions", new RoamingProfileStorage("TextEditor..Specific.RemoveUnnecessarySuppressionExclusions")}, // Used to contain %LANGUAGE% but deosn't have per-language value, keeping the wrong name for compat.
```

Comment:
Is this exposed in Visual Studio UI options? If not, how can its value change other than from `.editorconfig`?

Reply:
Good point. I don't see it.

Reply:
@mavasani Should this option be configurable in VS UI?

Reply:
Removed the storage for now, tracked by https://github.com/dotnet/roslyn/issues/66062